model/src/main/kotlin/org/digma/intellij/plugin/model/rest/insights/EndpointInfo.kt

```diff
@@ -7,11 +7,12 @@ import java.beans.ConstructorProperties
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class EndpointInfo
 @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
-@ConstructorProperties("route", "serviceName", "instrumentationLibrary", "codeObjectId","spanCodeObjectId")
+@ConstructorProperties("route", "serviceName", "instrumentationLibrary", "codeObjectId","spanCodeObjectId", "entrySpanCodeObjectId")
 constructor(
     val route: String,
     val serviceName: String,
     val instrumentationLibrary: String,
     val codeObjectId: String?,
-    val spanCodeObjectId:String
+    val spanCodeObjectId:String,
+    val entrySpanCodeObjectId:String?, // Nullable for BC
```

Comment:
can you link in this PR to the backend PR so we could see them linked?

Reply:
done